src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -1003,4 +1013,8 @@
   </xsl:attribute-set>
   -->
 
+  <!-- Provide enough space for two digit labels (needed in bibliography). -->
```

Comment:
I don't think you need the comment "needed in bibliography" since this is needed for any ordered list that exceeds 9 entries.